src-java/northbound-service/northbound-api/src/main/java/org/openkilda/northbound/dto/v2/haflows/HaFlowValidationResult.java

```diff
@@ -34,6 +34,5 @@
 @FieldDefaults(level = AccessLevel.PRIVATE)
 public class HaFlowValidationResult {
     boolean asExpected;
-    HaFlowDiscrepancy haFlowValidationResult;
```

Comment:
This is the last place where HaFlowDiscrepancy was used. Please remove HaFlowDiscrepancy too.

Reply:
fixed